ci/release/docker.go

```diff
@@ -0,0 +1,169 @@
+/*
+ * Copyright (C) 2019 The "MysteriumNetwork/node" Authors.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package release
+
+import (
+	"bufio"
+	"bytes"
+	"fmt"
+	"io/ioutil"
+	"os/exec"
+	"path/filepath"
+	"strings"
+
+	log "github.com/cihub/seelog"
+	"github.com/mysteriumnetwork/node/ci/env"
+	"github.com/mysteriumnetwork/node/ci/storage"
+	"github.com/mysteriumnetwork/node/logconfig"
+)
+
+const dockerImagesDir = "build/docker-images"
+const mystSnapshotsRepo = "mysteriumnetwork/myst-snapshots"
+
+// ReleaseDockerSnapshot uploads docker snapshot images to myst snapshots repo
+func ReleaseDockerSnapshot() error {
```

Comment:
> func name will be used as release.ReleaseDockerSnapshot by other packages, and that stutters; consider calling this DockerSnapshot (golint)

Reply:
Couldn't agree more from package perspective. BUT :)
```go
go run mage.go ReleaseDockerSnapshot
```
sounds more expressive as its not a normal function but mage action :|